src/lib/routes/admin-api/index.ts

```diff
@@ -112,6 +112,7 @@ class AdminApi extends Controller {
         this.app.use(
             '/invite-link',
             conditionalMiddleware(
+                // @ts-expect-error
                 () => config.flagResolver.isEnabled('publicSignup'),
```

Comment:
One of the strays caught by the PR - Maybe it's a feature flag we should clean up? Or, on the other hand, correctly add to the remaining feature flags?

Reply:
This should be removed.